modules/carousel/edit.html.twig

```diff
@@ -5,7 +5,7 @@
             'class': 'py-5 '~form.class.vars.data|default(''),
```

Comment:
As data-edit-class-default is **"sfs-carousel p-2"**, the class should be **'sfs-carousel py-5 '~form.class.vars.data|default('')**, to work exactly the same if you are loading the module from database or you are changing class from edit form.

Reply:
That's why we usually use a twig variable to "share" the value between changes ;)

Reply:
Done